portal/app/core/templates/dashboard.html

```diff
@@ -0,0 +1,103 @@
+{% extends "base.html" %}
+
+{% load static %}
+{% block partial_styles %}
+    <link rel="stylesheet" href="{% static 'css/dashboard.css' %}">
+    {% block dashboard-styles %}{% endblock dashboard-styles %}
+{% endblock partial_styles %}
+{% block main-content %}
+<div class="main-panel">
+    <div class="content-wrapper">
+        <div id="resources" class="container-fluid">
+            <div class="row">
+                <div class="col-md-11 grid-margin">
+                    <div class="card card-body row page-header">
+                        <ul class="nav">
+                            <li class="nav-item">
+                                <a class="nav-link" href="/">
+                                    <h4 class="page-title">Resources Information</h4>
+                                </a>
+                            </li>
+                            <li class="nav-item">
+                                <a class="nav-link icons w3-bar-item w3-button {% if resource == 'loads' %}current{% endif %}" href="/loads">
+                                    <image src="https://cdn3.iconfinder.com/data/icons/devices-and-communication-2/100/electricity_plug_female_american-512.png" height=30>
```

Comment:
Where did you get `image` tag from? It doesn't validate and it's forced map to `img`. Please only use `img`. Also, please quote all height and width values.

Reply:
done